Gemfile

```diff
@@ -1,49 +1,49 @@
 source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '5.0.0'
+gem 'rails', '7.0.2.3'
 
 # Use Puma as the app server
-gem 'puma', '3.7.0'
+gem 'puma'

```

Comment:
Gemfile.lock includes all the current versions, but I wonder if I should set them here too